arangod/RestHandler/RestIndexHandler.cpp

```diff
@@ -199,7 +205,102 @@ RestStatus RestIndexHandler::getIndexes() {
     tmp.add(StaticStrings::Error, VPackValue(false));
     tmp.add(StaticStrings::Code,
             VPackValue(static_cast<int>(ResponseCode::OK)));
-    tmp.add("indexes", indexes.slice());
+
+    if (ServerState::instance()->isCoordinator()) {
+      std::string ap =
+          absl::StrCat("Plan/Collections/", _vocbase.name(), "/",
+                       std::to_string(coll->planId().id()), "/indexes");
```

Comment:
The extra `std::to_string(number)` is not necessary here: absl::StrCat() handles that automatically.
```suggestion
                       coll->planId().id(), "/indexes");
```

Reply:
Cool, I did not know this. Will fix.